components/ws-manager/pkg/manager/pod_controller.go

```diff
@@ -34,8 +37,21 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 	err := r.Client.Get(context.Background(), req.NamespacedName, &pod)
 	if errors.IsNotFound(err) {
 		// pod is gone - that's ok
+		if pod, ok := r.Pods[req.NamespacedName]; !ok {
+			delete(r.Pods, req.NamespacedName)
+			queue := pod.Annotations[workspaceIDAnnotation]
+			log.Warnf("reconile: %v", queue)
```

Comment:
Consider removing this or changing to log.Debugf.
Otherwise, I guess the ws-manager would print lots of `reconile: xxx`, correct?

Reply:
ops, I forgot to remove it

Reply:
fixed